pages/overview/what-is-jsonschema.md

```diff
@@ -2,50 +2,49 @@
 section: docs
 title: What is JSON Schema?
 ---
-JSON Schema is a declarative language for annotating and validating JSON documents' structure, constraints, and data types. It provides a way to standardize and define expectations for JSON data.
+JSON Schema is a declarative language for annotating and validating JSON instances' structure, constraints, and data types. It provides a way to standardize and define expectations for JSON data.
```

Comment:
This is a key statement which it would be good to make sure the TSC sees and agrees on.

Reply:
I think "instances" is terminology precisely 0% of people understand who haven't read it somewhere in our docs (who seem likely to be 0% of people reading this page).

The consequence of that I think is:

* that word should be immediately interlinked to the glossary the first time it's used

but also

* I would personally not use it in the first sentence certainly. I'd use "JSON data".

Reply:
"define expectations for JSON data" is also confusing - data does not have expectations. People (developers) have expectations.
Perhaps "It provides a standard way to define what JSON data is valid for applications".

Reply:
"define people's expectations for JSON data"

Reply:
I agree that terms should be defined on first use. A glossary link is less than ideal in an intro, but better than nothing. The point is, I think these pages should use terms _consistently_ and avoid defining aliases which can raise questions (especially if those aliases are not defined in the glossary).
The glossary defines "instance" but not "document" (it probably should, and "JSON data" as well,
and perhaps distinguish between them all.)  I think many people think a "document" means streamable content in a file (or web resource), as opposed to in-memory data (i.e. file.json vs. a parsed JSON value in a data structure in a programming language), which is why I suggest not using "document" here. "JSON data" is better but it does add an alias for "instance"...

Reply:
I also agree with Julian that "instances" might be a bit vernacular-y at this point.

```suggestion
> JSON Schema is a declarative language for defining structure, constraints, and data types for JSON data.
```

Reply:
> structure, constraints, and data types

I feel like these words are all saying the same thing and using all of them is confusing. I'd just use "constraints" because JSON Schema isn't really about structure or data types. Those things are just a consequence of the constraints of the schema.